src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -35,11 +35,11 @@ public class FullNodeController : FeatureController
 
         public FullNodeController(
             ILoggerFactory loggerFactory,
-            IPooledTransaction pooledTransaction,
-            IPooledGetUnspentTransaction pooledGetUnspentTransaction,
-            IGetUnspentTransaction getUnspentTransaction,
-            INetworkDifficulty networkDifficulty,
-            IConsensusLoop consensusLoop,
+            IPooledTransaction pooledTransaction = null,
```

Comment:
You might also need to fix the part that calls this code, but this is for a later PR perhaps as now it wont crash.

Reply:
Yeah, will add the checks in this one.

Reply:
Fixed